src/main/java/browserview/BrowserComponent.java

```diff
@@ -123,12 +130,25 @@ private ChromeDriverEx createChromeDriver() {
         ChromeDriverEx driver = new ChromeDriverEx(options, isTestChromeDriver);
         WebDriver.Options manage = driver.manage();
         if (!isTestChromeDriver) {
+            setWindowSize(manage);
```

Comment:
Agreed, does `driverOptions` sound clearer? `manage` is taken from the chromedriver API, but if we're OK with changing the API then we can change it to `getDriverOptions`.